css-conditional-5/Overview.bs

```diff
@@ -1067,6 +1087,159 @@ Style Container Features</h3>
+		<dt><dfn>inline-start</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the inline-start edge.
+		<dt><dfn>block-end</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the block-end edge.
+		<dt><dfn>inline-end</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the inline-end edge.
+	</dl>
+
+<h4 id="snapped">
+Scroll snapping: the '@container/snapped' feature</h4>
+
+	<pre class="descdef mq">
+		Name: snapped
+		For: @container
+		Value: x | y | block | inline
+		Type: discrete
+	</pre>
+
+	The '@container/snapped' [=container feature=] queries whether a [=snap target=]
+	is snapped to its [=snap container=] in the given axis. It matches in the boolean
+	context if it is snapped in at least one of the directions.
+
+	<dl dfn-type=value dfn-for="@container/snapped">
+		<dt><dfn>x</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''x''
+			if the container is a horizontal [=snap target=] for its [=scroll container=]
+		<dt><dfn>y</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''y''
+			if the container is a vertical [=snap target=] for its [=scroll container=]
+		<dt><dfn>block</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''block''
+			if the container is a [=snap target=] for its [=scroll container=]
+			in the block direction of the [=snap container=].
+		<dt><dfn>inline</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''inline''
+			if the container is a [=snap target=] for its [=scroll container=]
+			in the inline direction of the [=snap container=].
+	</dl>
+
+<h4 id="overflowing">
+Sticky positioning: the '@container/overflowing' feature</h4>
+
+	<pre class="descdef mq">
+		Name: overflowing
+		For: @container
+		Value: top | right | bottom | left | block-start | inline-start | block-end | inline-end
+		Type: discrete
+	</pre>
+
+	The '@container/overflowing' [=container feature=] queries whether a
+	[=scroll container=] has [=scrollable overflow=] in the given direction for the
+	current scroll progress. The logical values map to physical based on the direction
+	and writing-mode of the [=query container=].
+
+	In the boolean context, the query matches if there is scrollable overflow in any
+	direction.
+
+	<dl dfn-type=value dfn-for="@container/overflowing">
+		<dt><dfn>top</dfn>
+		<dd>
+			The [=scroll container=] has [=scrollable overflow=] past the top edge.
+		<dt><dfn>right</dfn>
+		<dd>
+			The [=scroll container=] has [=scrollable overflow=] past the right edge.
+		<dt><dfn>bottom</dfn>
+		<dd>
+			The [=scroll container=] has [=scrollable overflow=] past the bottom edge.
+		<dt><dfn>left</dfn>
+		<dd>
+			The [=scroll container=] has [=scrollable overflow=] past the left edge.
+		<dt><dfn>block-start</dfn>
+		<dd>
+			The [=scroll container=] has [=scrollable overflow=] past the block-start edge.
```

Comment:
link the logical values to their [definitions](https://drafts.csswg.org/css-writing-modes-4/#logical-directions)?